src/content/reference/hi/p5/noSmooth.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: noSmooth
+module: आकार
+submodule: गुण
+file: src/core/shape/attributes.js
+description: >
+  <p>दांतेदार (उर्फ) किनारों के साथ कुछ विशेषताएं बनाता है।</p>
```

Comment:
sounds bit complex,  can you suggest some alternatives?

Reply:
Yeah sure, made it simpler.